Assets/Scripts/Tags/ClimbableTag.cs

```diff
@@ -3,16 +3,16 @@
 
 public class ClimbableTag : MonoBehaviour 
 {
-	[SerializeField] bool xMovement;
-	public bool XMovement
+	[SerializeField] bool _xMovement = true;
```

Comment:
I like this use case where the private variables accessed through an instance have underscores, but don't think usual private variables need an underscore.